prog/vm/nexus.rb

```diff
@@ -330,6 +330,8 @@ def before_run
     case host.sshable.cmd("common/bin/daemonizer --check prep_#{q_vm}")
     when "Succeeded"
       vm.nics.each(&:incr_setup_nic)
+      strand.stack[-1]["prep_done"] = true
```

Comment:
Not sure, but it might be a little more readable
```suggestion
      strand.stack.last["prep_done"] = true
```

Reply:
Since it will be removed in the next cleanup PR, you can ignore it